### PR TITLE
Fixed build error having to do with scala version

### PR DESCRIPTION
**Pull request by granders**
@ewencp @ijuma 
This fixes the build, but can you sanity check that I'm not doing anything unreasonable here?

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -81,6 +81,7 @@ fi
 # they default to, which should work ok for now.
 echo "Building Kafka"
 KAFKA_BUILD_OPTS=""
+SCALA_VERSION=2.11.7
```
Comment (ewencp) [inline build.sh]: Shouldn't this only be replacing if it's not already specified? The reason we had the flag in the first place was to allow you to specify the version to build by setting the env var before calling this script.

**Comment by ewencp** [comment]
@granders Aside from letting through user-specified values via that env var, LGTM

**Comment by granders** [comment]
Ok thanks @ewencp

File: build.sh [hunk 2]
```diff
@@ -81,6 +83,12 @@ fi
 # they default to, which should work ok for now.
 echo "Building Kafka"
 KAFKA_BUILD_OPTS=""
+
+if [ -z "$SCALA_VERSION" ]; then
+    echo "No scala version specified... using default value of $KAFKA_SCALA_VERSION"
+    SCALA_VERSION=$KAFKA_SCALA_VERSION
+fi
+
 if [ "x$SCALA_VERSION" != "x" ]; then
```
Comment (ijuma) [inline build.sh]: Can this check ever fail now?

Comment (granders) [inline build.sh]: Good point, no it cannot

**Pull request merged**